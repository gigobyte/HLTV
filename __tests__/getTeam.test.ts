import HLTV from '../src/'

function sleep(ms: number): Promise<void> {
  return new Promise((resolve) => setTimeout(resolve, ms))
}

const FULL = 4608
const EMPTY = 460

test('getTeam', async () => {
  await sleep(3000)
  expect(await HLTV.getPlayer({ id: FULL })).toMatchSnapshot()
  expect(await HLTV.getPlayer({ id: EMPTY })).toMatchSnapshot()
}, 30000)
