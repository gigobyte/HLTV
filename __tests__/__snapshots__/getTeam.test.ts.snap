// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getTeam 1`] = `
Object {
  "achievements": Array [],
  "age": undefined,
  "country": Object {
    "code": "SE",
    "name": "Sweden",
  },
  "facebook": undefined,
  "id": 4608,
  "ign": "frankdude",
  "image": undefined,
  "instagram": undefined,
  "name": "Frank Sjöström",
  "news": Array [],
  "statistics": undefined,
  "team": undefined,
  "teams": Array [],
  "twitch": undefined,
  "twitter": undefined,
}
`;

exports[`getTeam 2`] = `
Object {
  "achievements": Array [],
  "age": undefined,
  "country": Object {
    "code": "UA",
    "name": "Ukraine",
  },
  "facebook": undefined,
  "id": 460,
  "ign": "Alesinho",
  "image": undefined,
  "instagram": undefined,
  "name": "Aleksey Karamyshev",
  "news": Array [],
  "statistics": undefined,
  "team": Object {
    "id": 3094,
    "name": "Naruto",
  },
  "teams": Array [
    Object {
      "id": 3094,
      "leaveDate": 1614726000000,
      "name": "Naruto",
      "startDate": 1614726000000,
      "trophies": Array [],
    },
  ],
  "twitch": undefined,
  "twitter": undefined,
}
`;
